coco/request_forwarder.py

```diff
@@ -142,6 +207,7 @@ def __init__(self, blocklist_path: os.PathLike, timeout: int):
         self.queue_len = None
         self.queue_wait_time = None
         self.response_time = None
+        self._debug = debug
```

Comment:
Should we name this `debug_connections` to be consistent with the config option?  I realize this is inside the forwarder, but in theory `debug` could still mean different things in this context.

Reply:
Good point. Done.